import assert from "assert";


describe("Test", () => {
  it("Test1", async () => {
    try {
      // TODO
    } catch (error) {
      assert.fail(error);
    }
  })
})